### ISSUE TITLE
Crash when in map scene with connected controller

### ISSUE THREAD

**Issue by user670** [26F-Studio/Techmino#494]
Screenshot with crash information (*Image(s) Here*):
![image](https://user-images.githubusercontent.com/22617255/143690818-f7c1ca9a-32ad-4db3-a0aa-ce4fff7225bd.png)

If you can reproduce it, write the steps here. If you can't, try to describe the exactly time the game crash, like pressing which key or which button (*Details Here*)
- With a controller connected, no buttons other than "back" work on the main menu
- Use a mouse to enter the map, and it crashes even without any further inputs from any input devices
- ~~When I try playing in a mode (using simplistic mode or last played to bypass the map), analog triggers will only trigger once each; pressing the same trigger a second time doesn't work~~ I'm opening a new issue for this

~~Built from live repository contents, will test GH Action build later~~ Tested with GH Action build

**Comment by MrZ626** [comment]
New commit should fix this, need test again

**Comment by user670** [comment]
Nope
![image](https://user-images.githubusercontent.com/22617255/143726279-b67c5ef2-ef46-4c02-812e-3284bac7d8a5.png)

**Comment by MrZ626** [comment]
Btw, can you move the cursor with controllers?
Later the mode map will be removed so this issue will be invalid lol

**Comment by user670** [comment]
> Btw, can you move the cursor with controllers? Later the mode map will be removed so this issue will be invalid lol

No, I can't get that consistently working (somehow got that ONCE and ONLY ONCE and never again). Start key to click mouse works though
